### ISSUE TITLE
In high contrast Aquatic/ Desert mode, Selected date is not visually indicated as selected in windows.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#113657]
### What happened?

**User Impact:**
Low vision user who relies on high contrast settings will be impacted and unable to identify the selected date if in high contrast Aquatic/ Desert mode, Selected date is not visually indicated as selected in windows.

**Actual Result:**
In high contrast Aquatic/ Desert mode, Selected date is not visually indicated as selected in windows.

**Refer Attachment:**
In high contrast Aquatic & Desert mode, Selected date is not visually indicated as selected in windows.png
In high contrast Aquatic & Desert mode, Selected date is not visually indicated as selected in windows.mp4

https://github.com/user-attachments/assets/14914ea3-b8c6-4c5c-b82f-5733a984138e
<img width="1925" height="956" alt="Image" src="https://github.com/user-attachments/assets/ff8d9c15-bc6a-486c-a3aa-aa3a9d6fcb99" />

### What did you expect to happen?

In high contrast Aquatic/ Desert mode, Selected date should be visually indicated as selected in windows.

### Did this work before?

No

### How do we reproduce it?

**Pre-Requisites:**
Setting > Accessibility > Contrast Themes > Aquatic/ Desert mode

**Repro Steps:**
1. Open the URL: https://play.grafana.org/d/OhR1ID6Mk/table?orgId=1&from=now-6h&to=now&timezone=utc in edge browser in windows.
2. 'Grafana Table Dashboard' page will appear.
3. Press tab key to reach to the 'Last 6 Hours' control and activate it.
4. Press tab key to reach to 'From' and 'To' control and activate the date picker.
5. Verify whether in high contrast Aquatic/ Desert mode, Selected date is visually indicated as selected in windows or not. 

### Environment (with versions)?

Grafana: Grafana Cloud (fast)
OS: Windows 11 version 25H2 (OS Build 26220.7052)
Browser: Edge (Version 143.0.3650.7)
URL: https://play.grafana.org/d/OhR1ID6Mk/table?orgId=1&from=now-6h&to=now&timezone=utc

### Grafana platform?

I use Grafana Cloud

**Comment by msftedad** [comment]
Hi Team,
Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, As issue is still repro'ing, Any update here?

**Comment by msftedad** [comment]
Hi Team, As issue is still repro'ing, Any update here?
